### PR TITLE
#31 StrictXML should be able to look up XSDs from classpath

### PR DESCRIPTION
**Pull request by adko-pl**
@dmarkov Please find a reviewer

### CODE REVIEW

**Comment by dmarkov** [comment]
Thanks, let me find someone who can review this pull request

**Comment by dmarkov** [comment]
@longtimeago review this one please

File: src/main/java/com/jcabi/xml/ClasspathInput.java [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+        return null;
+    }
+
+    @Override
+    @SuppressWarnings("PMD.AvoidThrowingRawExceptionTypes")
+    public String getStringData() {
+        try {
+            return IOUtils.toString(this.stream);
+        } catch (final IOException exception) {
+            throw new RuntimeException("Unable to read input", exception);
```
Comment (longtimeago) [inline src/main/java/com/jcabi/xml/ClasspathInput.java]: why throw RuntimeException?
Throw IllegalArgumentException and remove SuppressWarnings

**Comment by longtimeago** [comment]
@adko-pl See one remark above. Also it would be good to test negative case, where resource is not in classpath.

**Comment by adko-pl** [comment]
@longtimeago added your comments; travis build fails due to #60

**Comment by longtimeago** [comment]
@adko-pl Thank you!

**Comment by longtimeago** [comment]
@rultor good to merge

**Comment by longtimeago** [comment]
@yegor256 need your approve here

**Comment by yegor256** [comment]
@rultor merge

**Comment by dmarkov** [comment]
@longtimeago Thanks so much! Your account was topped up for **18 mins** (transaction ID is `52539582`); you're getting extra minutes here (c=3); +18 to your rating, your total score is [+2341](http://www.netbout.com/b/35024?open=rating)

**Comment by dmarkov** [comment]
@rultor please deploy
